src/ui.cpp

```diff
@@ -823,17 +823,13 @@ bool uilist::scrollby( const int scrollby )
         if( fselected >= recmax ) {
             fselected = 0;
         }
-        int x = 0;
         for( size_t i = 0; i < fentries.size(); ++i ) {
```

Comment:
Not in the scope of this PR, but I'd suggest to look into merging these two nearly identical loops.